release_notes/topics/8_0_0.adoc

```diff
@@ -45,6 +45,18 @@ In this release, we added initial support for W3C Web Authentication (WebAuthn).
 however we are working on further improvements in this area. Thanks to https://github.com/tnorimat[tnorimat] for the contribution. Also thanks to
 https://github.com/ynojima[ynojima] for the help and feedback.
 
+== Support for password-less authentication, multi-factor authentication and multiple credentials per user
+
+With the arrival of W3C Web Authentication support, we've refined the authentication flow system to be able to allow a user to select which authentication method he
+wishes to use during his login (for example, to choose between his OTP credential and his WebAuthn credential). The new mechanisms also allow an administrator to
+craft flows for password-less login, for example just using WebAuthn as an authentication method. Please note that with these changes, any custom authentication
+flow you have created may need to be adapted to the new flow logic.
+
+As a result of these changes, it is now also possible for users to have multiple OTP devices and multiple WebAuthn devices. The same system that allows a user
```

Comment:
Shorten this sentence.   Change "it is now also possible for users to have"   to  "users can now have"